assembly/runtime/env/imports.ts

```diff
@@ -159,17 +172,4 @@ export declare namespace env {
   //@ts-ignore
   @external("env", "storage_has_key")
   export function storage_has_key(key_len: u64, key_ptr: u64) : u64;
-  //@ts-ignore
-  @external("env", "storage_iter_prefix")
-  export function storage_iter_prefix(prefix_len: u64, prefix_ptr: u64) : u64;
-  //@ts-ignore
-  @external("env", "storage_iter_range")
-  export function storage_iter_range(start_len: u64, start_ptr: u64, end_len: u64, end_ptr: u64) : u64;
-  //@ts-ignore
-  @external("env", "storage_iter_next")
-  export function storage_iter_next(iterator_id: u64, key_register_id: u64, value_register_id: u64) : u64;
-  //@ts-ignore
-  // Function for the injected gas counter. Automatically called by the gas meter.
-  @external("env", "gas")
```

Comment:
This is for testing purposes so in unit tests you can get gas usage.  Maybe we should move it else where.

Reply:
not sure.  should i put it back?

Reply:
Yeah you could move it to `assembly/bindgen.ts`

Reply:
```
ERROR AS212: Decorator '@external' is not valid here.

 @external("env", "gas")
 ~~~~~~~~~~~~~~~~~~~~~~~
 in ../assembly/bindgen.ts(10,0)
```

Reply:
we agreed to skip this